vtm.js

```diff
@@ -190,28 +252,103 @@ const app = Vue.createApp({
           },
         ],
       },
+      disciplines:{
+        id: "Disciplines",
+        resource: [0,1,1,0,0,0],
+        data: [
+          {
+            id: 'Animalism',
+            value: 0,
+          },
+          {
+            id: 'Auspex',
+            value: 0,
+          },
+          {
+            id: 'Blood Sorcery',
+            value: 0,
+          },
+          {
+            id: 'Celerity',
+            value: 0,
+          },
+          {
+            id: 'Dominate',
+            value: 0,
+          },
+          {
+            id: 'Fortitude',
+            value: 0,
+          },
+          {
+            id: 'Obfuscate',
+            value: 0,
+          },
+          {
+            id: 'Potence',
+            value: 0,
+          },
+          {
+            id: 'Presence',
+            value: 0,
+          },
+          {
+            id: 'Protean',
+            value: 0,
+          },
+        ],
+      },
     };
   },
+  
   template: `
-  <stat-section 
+  <character-info
+    :bio="biography"
+    :clans="clans">
+  </character-info>
+  <attribute-section 
     :stats="attributes"
-    @stat-section-change="$event[0].value=$event[1]"
-    >
-  </stat-section>
-  <stat-section 
+    @stat-section-change="$event[0].value=$event[1]">
+  </attribute-section>
+  <skill-section 
     :stats="skills"
-    @stat-section-change="$event[0].value=$event[1]"
-    >
-  </stat-section>`
+    :distributions="skillDistributions"
+    @stat-section-change="$event[0].value=$event[1]">
+  </skill-section>
+  <discipline-section 
+    v-if="biography.clan"
+    :stats="disciplines"
+    :clan="biography.clan"
+    @stat-section-change="$event[0].value=$event[1]">
+  </discipline-section>`
 });
 
-/** Displays entire section of attributes or skills
```

Comment:
nemal by tam komentar zostat?

Reply:
... musim sa najprv pozret na mixin, co vlastne robi a preco ho tam davas

Reply:
aha, to je jakoze nieco co sa da pouzit v inej komponente

Reply:
nema to tam zostat... som to rozdeloval, a komentar zmenil, jedno na skills, druhe attributy